src/GusApi/Util/BulkReportResponseDecoder.php

```diff
@@ -27,7 +29,7 @@ public static function decode(GetBulkReportResponseRaw $bulkReportResponseRaw):
             }
 
             return $regons;
-        } catch (\Exception $e) {
```

Comment:
Instead of skipping `$e`, I would pass it to `InvalidServerResponseException` as third argument (`$previous`) so that it can be handled nicely - see https://www.php.net/manual/en/exception.getprevious.php

Reply:
For now i've just run default rector php8 ruleset probably it was part of this. I will try to check it later if i will have time ;)